### DISCUSSION TITLE
Two Quick Questions with Article Filters, Full Text Match or Contains Match?

### DISCUSSION THREAD

**Discussion by quella** [FreshRSS/FreshRSS#6105] [Q&A] [answered]
Just two quick questions regarding feed filters.  

If using FREETEXT in a feed filter, does this span all elements of the article such as title, author, etc. or does it only search the text within the article?  

Secondly, is the filter a "contains" or "exact match"?  For example, if I had an article with the text "this is the motherofall articles", would the label match using freetext of "mother" or would I have to use "motherofall" for an exact match?    

I would exclude the quotes in the above free text, I used them here in the question for clarity.

**Comment by Alkarex** [comment, marked as answer]
If you search a free-text expression, it will search in title and content of articles.
You can be explicit and search only in title, or search in authors, tags, etc. with specific syntax: https://freshrss.github.io/FreshRSS/en/users/10_filter.html#with-the-search-field

* `hello world` will search for content or title containing the sub-string `hello` AND also containing the sub-string `world`. This means that it will match an article containing `wordddd helloooo`
* `"hello world"` with quotes will search for content or title containing the sub-string `hello world`, meaning that it will match an article containing `hhhhello worlddddd`

**Comment by quella** [comment]
@Alkarex Thank you for the great information regarding filters.  I assume the freetext option was only in the article and an exact match.  I can do some cleaning of my many filters to help limit the number of them.  Glad that it is a CONTAINS by default unless explicitly specific by the use of quotes.  Thank you sir.
